tests/bearlib/aspects/ClassTest.py

```diff
@@ -56,3 +57,21 @@ class SubAspect:
         assert (str(exc.value) == "<class 'str'> is not an aspectclass or "
                 'an instance of an aspectclass')
         assert issubaspect(SubAspect('Python'), RootAspect)
+
+
+class AspectlistClassTest:
+
+    def test__init__(self):
+        with pytest.raises(TypeError) as exc:
+            aspectlist(['String'])
+        assert (str(exc.value) == "'String' is not an aspectclass or "
+                'an instance of an aspectclass')
+
+    def test__contains__(self):
+        list_of_aspect = aspectlist(
+            [Metadata.CommitMessage.Shortlog, Metadata.CommitMessage.Body])
```

Comment:
Yeah we also don't check the format of any new aspect-class addition in the aspect-class tree.